ext/intl/formatter/formatter.stub.php

```diff
@@ -92,6 +92,16 @@ class NumberFormatter
      * @cvalue UNUM_ROUND_UP
      */
     public const ROUND_UP = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue UNUM_ROUND_DOWN
+     */
+    public const ROUND_TOWARD_ZERO = UNKNOWN;
```

Comment:
please add type declaration to new class constants

Reply:
I will add. I'm just waiting [this PR](https://github.com/php/php-src/pull/12360) to be merged first.

Reply:
Done 👍🏾